auth2client/src/components/Signup.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Signup extends Component {
+    state = {
+        username: "",
+        password: "",
+        department: ""
+    };
+
+    render() {
+        return (
+        <form onSubmit={this.handleSubmit}>
+
+            <div className="form-username">
+            <label htmlFor="username">Username</label>
+            <input
+                name="username"
+                value={this.state.username}
+                onChange={this.handleChange}
+                type="text"
+            />
+            </div>
+
+            <div>
+            <label htmlFor="password">Password</label>
+            <input
+                name="password"
```

Comment:
It is a good idea to have a second password input field to test if the user's two password's match. This can be done by both password input fields to state and then checking if password1 === password2 and if not, throwing an error that the user can then see.